components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java

```diff
@@ -179,27 +179,26 @@ public DataSection phenotypeBody(Patient patient) throws Exception
             sectionFeatureLookup = this.phenotypeHelper.getSectionFeatureTree();
         }
 
-        Boolean lastStatus = false;
+        Boolean lastStatus = null;
         String lastSection = "";
         for (Feature feature : sortedFeatures) {
             x = 0;
 
-            if (bothTypes && lastStatus != feature.isPresent()) {
-                lastStatus = feature.isPresent();
+            if (present.contains("positive") || present.contains("negative")) {
                 lastSection = "";
-                DataCell cell = new DataCell(lastStatus
+                DataCell cell = new DataCell(feature.isPresent()
                     ? this.translationManager.translate("yes")
                     : this.translationManager.translate("no"),
                     x, y);
-                if (!lastStatus) {
+                if (bothTypes && lastStatus != null && lastStatus != feature.isPresent()) {
```

Comment:
you can get rid of the `bothTypes` variable completely, since if there are no "both types" the check `lastStatus != null && lastStatus != feature.isPresent()` will never be true. Same in the other method below